web/frontend/src/components/slam/index.svelte

```diff
@@ -133,7 +133,7 @@ const refresh2d = async () => {
       if (reloadMap || pointcloud === undefined) {
         let response;
         [pointcloud, response] = await Promise.all([
-          slamClient.getPointCloudMap(),
+          slamClient.getPointCloudMap(!reloadMap),
```

Comment:
Add a comment saying we're getting the edited map by passing in a true bool.

Reply:
added a comment. Also changed the variable name to make this behavior more obvious